crypto/src/public_key.rs

```diff
@@ -68,6 +74,7 @@ impl From<PublicKey> for Hash {
             PublicKey::Ed25519(tz1) => tz1.into(),
             PublicKey::Secp256k1(tz2) => tz2.into(),
             PublicKey::P256(tz3) => tz3.into(),
+            PublicKey::Bls(tz4) => tz4.into(),
```

Comment:
I haven't seen a test for this function.

Reply:
Good spot! Have added a test for all the pk_hash variants

Reply:
Thanks. I've reviewed the test, and checked the pkh matches the one from Octez; everything LGTM.